<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InGame</class>
 <widget class="QWidget" name="InGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#InGame{background-color:black;}</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>780</y>
     <width>1155</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #5D5D5D, stop: 0.4 #2F2F2F, stop:1 #1C1C1C)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame {border:4px ridge #2F2F2F; border-radius:25px; background-image:url(&quot;&quot;); background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #5D5D5D, stop: 0.4 #2F2F2F, stop:1 #1C1C1C)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>791</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buyButton">
        <property name="text">
         <string>Buy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buildButton">
        <property name="text">
         <string>Build</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="upgradeButton">
        <property name="text">
         <string>Upgrade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sellButton">
        <property name="text">
         <string>Sell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="destroyButton">
        <property name="text">
         <string>Destroy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="moneyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel{background:transparent;color:rgb(0, 220, 0)}</string>
     </property>
     <property name="text">
      <string>58673 $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>10</y>
       <width>46</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1C1C1C, stop: 0.4 #2F2F2F, stop:1 #5D5D5D)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_5 {border:4px ridge #2F2F2F; border-radius:25px; background-image:url(&quot;&quot;); background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1C1C1C, stop: 0.4 #2F2F2F, stop:1 #5D5D5D)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>761</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="ownerFieldButton">
        <property name="text">
         <string>My fields</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="allFieldButton">
        <property name="text">
         <string>All fields</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="defaultFieldButton">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{color:white;background:transparent;}</string>
     </property>
     <property name="text">
      <string>00 : 00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>90</y>
     <width>205</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_6 {border:0px ridge #2F2F2F; border-radius:0px; background-image:url(&quot;&quot;); background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #5D5D5D, stop: 0.4 #2F2F2F, stop:1 #1C1C1C)}
QLabel{color:white;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QLabel" name="fieldTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font:bold 22pt;}</string>
    </property>
    <property name="text">
     <string>Field infos</string>
    </property>
   </widget>
   <widget class="QLabel" name="buildingTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font:bold 22pt;}</string>
    </property>
    <property name="text">
     <string>Building infos</string>
    </property>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_6</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
